sectionproperties/pre/dxf.py

```diff
@@ -0,0 +1,34 @@
+import pathlib
+from shapely.geometry import Polygon, MultiPolygon
+from sectionproperties.pre.sections import Geometry, CompoundGeometry
+
+
+def load_dxf(dxf_filepath: pathlib.Path):
+    """
+            Import any-old-shape in dxf format for analysis.
+            Code by aegis1980 and connorferster
+        """
+    c2s = None
+    try:
+        import cad_to_shapely as c2s  # type: ignore
+    except ImportError as e:
+        print(e.message)
+        print("To use 'from_dxf(...)' you need to 'pip install cad_to_shapely'")
+        return
+
+    if not dxf_filepath.exists():
+        raise ValueError(f"The filepath does not exist: {dxf_filepath}")
+
+    # TODO avoid step of making a temp file locally
```

Comment:
Doesn't look like we're writing a temp file now. Does this function actually save a temp file, or is this comment outdated?

Reply:
Yeah, I was not sure what's going on there. It does not seem like we are writing a temp file so I can remove that.